css/index.css

```diff
@@ -104,14 +179,57 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    /* background:teal; */
     margin: 20px 2.5%;
+    font-size: 0.25rem;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
-}
 
```

Comment:
Nice work allowing the parent container to style the individual boxes.  You only override the color for the individual boxes.